src/XenCert/StorageHandler.py

```diff
@@ -54,6 +54,18 @@ def report(predicate, condition):
 # Hardcoded time limit for Functional tests in hours
 timeLimitFunctional = 4
 
+def retval_judge(retval, exception, checkpoint=None, point=None):
+    if not retval:
+        raise Exception(exception)
+    if checkpoint:
+        checkpoint += point
+    return checkpoint
```

Comment:
How about below?
···
 check_result(retval, exception, checkpoint=0, point=1):
    if not retval:
        raise Exception(exception)
    return checkpoint + point
···

Reply:
ok, changed.